static/src/javascripts/projects/common/views/experiments/measure-understanding.html

```diff
@@ -0,0 +1,12 @@
+<div class="understanding">
+    <div class="understanding__text">Do you feel better informed as a result of reading this piece?</div>
+    <button class="understanding__choice js-button" data-value="yes">
```

Comment:
If screenreader people would tab through the page, all they get with this approach would be `Yes` and `No` labels.

I'd rather suggest converting this into `input[type="radio"]`, wrapped into a fieldset. What do you think about it?